vmdk_plugin/utils/log_formatter/log_formatter.go

```diff
@@ -83,3 +86,51 @@ func (f *VmwareFormatter) appendKeyValue(b *bytes.Buffer, key string, value inte
 
 	b.WriteByte(' ')
 }
+
+// init log with passed logLevel (and get config from configFile if it's present)
+// returns True if using defaults,  False if using config file
+func LogInit(logLevel *string, logFile *string, configFile *string) bool {
+	usingConfigDefaults := false
+	c, err := config.Load(*configFile)
+	if err != nil {
+		if os.IsNotExist(err) {
+			usingConfigDefaults = true // no .conf file, so using defaults
+			c = config.Config{}
+			config.SetDefaults(&c)
+		} else {
+			panic(fmt.Sprintf("Failed to load config file %s: %v",
+				*configFile, err))
+		}
+	}
+
+	path := c.LogPath
+	if path == "" {
+		path = config.DefaultLogPath
```

Comment:
Thanks for catching this.
I lost this change when refactoring this LogInit function.
will update it.